lerobot/common/robot_devices/motors/dynamixel.py

```diff
@@ -0,0 +1,588 @@
+                f"Read failed due to communication error on port {self.port} for group_key {group_key}: "
+                f"{self.packet_handler.getTxRxResult(comm)}"
+            )
+
+        values = []
+        for idx in motor_ids:
+            value = self.group_readers[group_key].getData(idx, addr, bytes)
+            values.append(value)
+
+        values = np.array(values)
+
+        # TODO(rcadene): explain why
+        if data_name in CONVERT_UINT32_TO_INT32_REQUIRED:
+            values = uint32_to_int32(values)
+
+        if data_name in CALIBRATION_REQUIRED:
+            values = self.apply_calibration(values, motor_names)
+
+        if data_name in CONVERT_POSITION_TO_ANGLE_REQUIRED:
+            values = motor_position_to_angle(values)
+
+        # log the number of seconds it took to read the data from the motors
+        delta_ts_name = get_log_name("delta_timestamp_s", "read", data_name, motor_names)
+        self.logs[delta_ts_name] = time.perf_counter() - start_time
+
+        # log the utc time at which the data was received
+        ts_utc_name = get_log_name("timestamp_utc", "read", data_name, motor_names)
+        self.logs[ts_utc_name] = capture_timestamp_utc()
+
+        return values
+
+    def write(self, data_name, values: int | float | np.ndarray, motor_names: str | list[str] | None = None):
+        if not self.is_connected:
+            raise ValueError(f"DynamixelMotorsBus({self.port}) is not connected. You need to run `motors_bus.connect()`.")
+
+        start_time = time.perf_counter()
+
+        if motor_names is None:
+            motor_names = self.motor_names
+
+        if isinstance(motor_names, str):
+            motor_names = [motor_names]
+
+        if isinstance(values, (int, float, np.integer)):
+            values = [int(values)] * len(motor_names)
+
+        values = np.array(values)
+
+        motor_ids = []
+        models = []
+        for name in motor_names:
+            motor_idx, model = self.motors[name]
+            motor_ids.append(motor_idx)
+            models.append(model)
+
+        if data_name in CONVERT_POSITION_TO_ANGLE_REQUIRED:
+            values = motor_angle_to_position(values)
+
+        if data_name in CALIBRATION_REQUIRED:
+            values = self.revert_calibration(values, motor_names)
+
+        # TODO(rcadene): why dont we do it?
+        # if data_name in CONVERT_INT32_TO_UINT32_REQUIRED:
+        #     values = int32_to_uint32(values)
+
+        values = values.tolist()
+
+        # TODO(rcadene): assert all motors follow same address
+        addr, bytes = self.model_ctrl_table[model][data_name]
+        group_key = get_group_sync_key(data_name, motor_names)
+
+        init_group = data_name not in self.group_readers
+        if init_group:
+            self.group_writers[group_key] = GroupSyncWrite(
+                self.port_handler, self.packet_handler, addr, bytes
+            )
+
+        for idx, value in zip(motor_ids, values, strict=False):
```

Comment:
Can they have different lengths?
```suggestion
        for idx, value in zip(motor_ids, values, strict=True):
```

Reply:
Added suggestion